packages/analytics-ma-browser/src/browser-ma-client.ts

```diff
@@ -0,0 +1,34 @@
+import { createInstance as createBrowserInstance } from '@amplitude/analytics-browser';
+import { returnWrapper } from '@amplitude/analytics-core';
+import { BrowserClient, BrowserOptions } from '@amplitude/analytics-types';
+import {
+  pageViewTrackingPlugin,
+  Types as PageViewTrackingPluginTypes,
+} from '@amplitude/plugin-page-view-tracking-browser';
+import { webAttributionPlugin, Types as WebAttributionPluginTypes } from '@amplitude/plugin-web-attribution-browser';
+
+export type BrowserMaOptions = BrowserOptions & { attribution: WebAttributionPluginTypes.AttributionPluginOptions } & {
+  trackPageViews: PageViewTrackingPluginTypes.PageTrackingBrowserOptions | boolean;
+};
+
+export const createInstance = (): BrowserClient => {
+  const client = createBrowserInstance();
+
+  const init = async (apiKey: string, userId?: string, options?: BrowserMaOptions) => {
+    if (!options?.attribution?.disabled) {
+      await client.add(webAttributionPlugin(client, { ...options?.attribution })).promise;
+    }
+    if (options?.trackPageViews) {
+      const pageViewTrackingPluginOptions = typeof options.trackPageViews === 'boolean' ? {} : options.trackPageViews;
+      await client.add(pageViewTrackingPlugin(client, pageViewTrackingPluginOptions)).promise;
+    }
```

Comment:
Where will the documentation and examples for these options live?

Reply:
I add an example to the README and it should be documented on the dev docs.